scout/server/blueprints/institutes/templates/overview/causatives.html

```diff
@@ -38,21 +38,11 @@
 {% block scripts %}
   {{ super() }}
   {{ db_table_external_scripts() }}
-  <script type="text/javascript">
+  <script>
     var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
     var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
       return new bootstrap.Tooltip(tooltipTriggerEl)
     })
-
-    $(document).ready(function() {
-      $('#causatives_table').DataTable( {
-          paging: false,
-          dom: 'Bfrtip',
-          buttons: [
-              'copyHtml5',
-              'excelHtml5',
-          ]
-      } );
-    } );
   </script>
+  <script src="{{ url_for('overview.static', filename='variants_list_scripts.js') }}"></script>
```

Comment:
Again, just for learning, why a static script js file vs a jinja macro js script? Same difference?

Reply:
It made sense for 2 reasons:
- It contains static code (while jinja macros ofter are used to populate content according to the params that you pass to them)
- JS code is included in static files in many other blueprints, so this would be consequent